src/config/urls.ts

```diff
@@ -1,4 +1,12 @@
+export const ENV_DEVELOPMENT = 'DEVELOPMENT'
+export const ENV_PRODUCTION = 'PRODUCTION'
+export const ENV_STAGING = 'STAGING'
+
 const readEnv = (name: string): string | undefined => {
+  const env = window.localStorage.getItem('env')
```

Comment:
Let's move this and `process.env.REACT_APP_ENV` to some separate file where we clearly define what each environment means. It can be in `src/config/envs.ts`:
```ts
export const BUILD_ENV = process.env.REACT_APP_ENV || 'production'
export const TARGET_DEV_ENV = window.localStorage.getItem('target_env') || 'staging'
```